src/clientmgr.c

```diff
@@ -67,21 +63,21 @@ void print_client(struct client *client) {
 		printf(")\n");
 	}
 
-	for (int i = 0; i < VECTOR_LEN(client->addresses); i++) {
-		struct client_ip *e = &VECTOR_INDEX(client->addresses, i);
+	for (int i = 0; i < VECTOR_LEN(client->ip_addresses); i++) {
+		struct client_ip_address *client_ip_address_iterator = &VECTOR_INDEX(client->ip_addresses, i);
```

Comment:
*client_ip_address_iterator* is much too long, and it is not an iterator. I think *e* is fine here, as it's short for "element"; *addr* and *ip* would also be options.